driver/pace/driver/run.py

```diff
@@ -310,6 +266,41 @@ def _initialize_dycore_state(
         return dycore_state
 
 
+@dataclasses.dataclass
+class TranslateConfig(InitializationConfig):
```

Comment:
The Translate was meant for coming from the Translate objects, where the state is pre-filled by the test infrastructure... it can be used with dycore or only or dycore+physics, it's based on the namelist settings. I agree we should regression test the SerialboxConfig, but having this be called as part of the test requires passing through the namelist path, which no other test uses or has access to. I can make it an optional argument, but it adds complication for a single use-case. I also think it is useful to be able to setup configuration in a way where you provide the initialized dycore_state. Instead of `TranslateConfig`, it could be something like `PredefinedDycoreStateConfig`. We could do both, call the driver through SerialboxConfig and this other config, and assert they produce the same results. Mybe we should discuss in slack.

Reply:
> I also think it is useful to be able to setup configuration in a way where you provide the initialized dycore_state.

This might be the way to go, I realized a problem with what I suggested is that this class initializes the grid from serialized data, while the existing config creates the grid with MetricTerms. If we do keep it this route, then we could name this more generally like you suggested, since the prepared dycore_state doesn't need to come from the translate infrastructure. I'd say `PredefinedStateConfig` is more accurate since you're also pre-defining the grid.

Not too relevant since it seems we're both leaning to keeping it like this, The driver test is pretty fundamentally different from the other tests since it's the topmost level test. It could be reasonable to write a separate test for it, instead of parameterizing over the same serial/parallel translate test used for every other case. There aren't too many serialbox tests where we'd run into this.